ross/fluid_flow/fluid_flow.py

```diff
@@ -266,6 +269,41 @@ def external_radius_function(self, gama):
 
         return radius_external, xre, yre
 
+    def get_rotor_load(self):
+        """Returns the load applied to the rotor.
```

Comment:
I think we should use the [numpydoc docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#sections).